src/main/java/racing/domain/CarResultVO.java

```diff
@@ -0,0 +1,29 @@
+package racing.domain;
+
+public class CarResultVO implements Comparable<CarResultVO> {
```

Comment:
```suggestion
public class CarResultVo implements Comparable<CarResultVo> {
```
도메인 패키지에 존재하는 게 맞을까요?
해당 객체의 패키지 위치를 고려해 주세요~ 🙇